packages/web-react/src/components/Header/README.md

```diff
@@ -1,119 +1,399 @@
-# Header
+# Header and HeaderDialog
 
-Header wraps top-level navigation and branding.
+The Header is a highly variable and customizable component. It comes in several
+design variants and provides a handful of building blocks you can use to achieve
+your specific design goals.
 
-## Simple Header
+The Header and HeaderDialog are a composition of several subcomponents:
 
 ```jsx
-<Header id="simple">
-  <Link href="/">
-    <Logo />
+<Header />
+  <HeaderMobileActions />
```

Comment:
The nesting here looks weird as each element is self-closing, so why is there any indentation?

Reply:
This is only structure